lua/pug/modules/lagdetection.lua

```diff
@@ -77,7 +91,7 @@ u.addHook("Tick", "LagDetection", function()
 	if halt then return end
 
 	local sysTime = SysTime()
-	local isReady = ( sample.mean == 0 )
+	local isReady = ( sample.mean ~= 0 )
```

Comment:
Fairly certain this is meant to be a `~=` because otherwise, we'd never make it past the `if not isReady` check (because `sample.mean` would never be 0 after the first iteration, which I believe was the original intention with this check).

Reply:
It should be making it past the `if not isReady` after the first iteration (by failing on that check). Because the check it self has a `return` statement in there, I've done this to let the server think for a bit before doing anything with the lag detection. Because when testing with APG on start up, I was getting tons of false positives for lag detection... so I decided to give it some breathing room for that start up time.